src/middlewared/middlewared/plugins/network.py

```diff
@@ -1094,74 +1094,74 @@ async def _common_validation(self, verrors, schema_name, data, itype, update=Non
                     'Failover needs to be disabled to perform network configuration changes.'
                 )
 
-                # have to make sure that active, standby and virtual ip addresses are equal
-                active_node_ips = len(data.get('aliases', []))
-                standby_node_ips = len(data.get('failover_aliases', []))
-                virtual_node_ips = len(data.get('failover_virtual_aliases', []))
-                are_equal = active_node_ips == standby_node_ips == virtual_node_ips
-                if not are_equal:
-                    verrors.add(
-                        f'{schema_name}.failover_aliases',
-                        'You must specify an active, standby and virtual IP address.'
-                    )
-
-                if not update:
-                    failover_attrs = set(
-                        [k for k, v in validation_attrs.items() if k not in ('mtu', 'ipv4_dhcp', 'ipv6_auto')]
-                    )
-                    configured_attrs = set([i for i in failover_attrs if data.get(i)])
```

Comment:
I dont quite understand why this was moved out of the if block?

Reply:
Everything had to be moved outside of the `if ha_configured and not failover['disabled']` statement or none of this validation will run.

Reply:
Ohhh, I thought it was outside `if not await self.middleware.call('failover.licensed')` block